oc/oc-netobserv-flows

```diff
@@ -2,11 +2,14 @@
 source "${BASH_SOURCE%/*}/network-observability-cli-resources/functions.sh"
 
 # interface filter such as 'br-ex'
-filter=$1
+filter=""
 
-if [ -z "$filter" ]
+if [ -z "${1:-}" ]
 then
-  filter=""
+  echo "Filters not set"
+else
+  echo "Filters set as $1"
+  filter=$1
```

Comment:
Why u set it again?

Reply:
I don't set it anymore:
- Line 5 to avoid bash to fail if not set
- Line 7 checks if the variable exists without failing
- Line 12 set the variable (only if it exists)

Reply:
/lgtm